jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml

```diff
@@ -226,6 +226,101 @@
             image-new: 'gci'
             version-env: ''
 
+# This is the job definition for GCE upgrade tests for GCI and between container-vm and GCI.
```

Comment:
I think you can avoid creating a new job group by reusing the one defined [here](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml#L344).

That way you only need to copy the last three jobs [here](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml#L452).
However there will be 4 jobs instead of 3
gci-gci
gci-container_vm
container_vm-gci
container_vm-container_vm